ietf/nomcom/views.py

```diff
@@ -207,6 +208,8 @@ def private_index(request, year):
                 elif action == "set_as_pending":
                     nominations.update(state='pending')
                     messages.success(request,'The selected nominations have been set as pending')
+                elif action == 'email':
+                    mailto = ','.join([np.nominee.email.name_and_email() for np in nominations])
```

Comment:
I suggest just email for now (or take on hardening name_and_email to properly quote name if it has commas in it. We have names like "foo bar, Ph.D." and things much less sensical have made it into the database.